content/d-products/1-cocoon.md

```diff
@@ -6,8 +6,8 @@ weight: 1
 
 ### Alvyl Cocoon
 
-Alvyl Cocoon is a product offering for Internet of Things (IoT), built by Alvyl Consulting.
+Alvyl Cocoon is an Internet of Things (IoT) product.
 
-Cocoon provides cloud hosted services for acquisition of real-time data from Industrial Machines and managing IoT Edge Devices, at Scale.
+Cocoon provides Edge and Cloud hosted services for acquisition of real-time data from Industrial Machines and managing IoT Edge Devices, at Scale.
```

Comment:
`Scale` -> `scale`

Reply:
I think if I try and bold them, I don't have to rely on capital `Scale`. That way I can stay coherent with grammer, and highlight.

Thanks for the comments, and hence indirectly the idea. :-)